setup.py

```diff
@@ -43,6 +42,16 @@
         name=name,
         version=package_info["VERSION"],
         description="Insights Application Programming Interface",
+        author_email="insights@redhat.com",
+        license="Apache 2",
+        url="https://access.redhat.com/insights",
+        classifiers=[
+            "Development Status :: 5 - Production/Stable",
+            "License :: OSI Approved :: Apache Software License",
```

Comment:
I wasn't sure what the "official" terms where. I found this one [here](https://pypi.python.org/pypi?%3Aaction=list_classifiers), but I didn't find an official list of license terms.